zenoh-kotlin/src/commonMain/kotlin/io/zenoh/protocol/Reliability.kt

```diff
@@ -12,7 +12,7 @@
 //   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
 //
 
-package io.zenoh.subscriber
+package io.zenoh.protocol
```

Comment:
`io.zenoh.protocol` should not be exposed to the API. In Rust there is a simple rexport of the type that appears as if it was defined in the Rust crate.

`CongestionControl`, `Reliability`, `Priority` should be part of a `io.zenoh.qos` package.